modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -562,6 +562,46 @@ def test_loc_multi_index():
     df_equals(modin_df.loc[modin_df.index[:7]], pandas_df.loc[pandas_df.index[:7]])
 
 
+def test_loc_multi_index_with_tuples():
+    arrays = [
+        ["bar", "bar", "baz", "baz", "foo", "foo", "qux", "qux"],
+        ["one", "two", "one", "two", "one", "two", "one", "two"],
+    ]
+    columns = pd.MultiIndex.from_tuples(zip(*arrays), names=["a", "b"])
+    data = [np.random.randint(10, 50, len(columns)) for _ in range(30)]
```

Comment:
We shouldn't use random data in tests without seeding `np`. Anyway, can we use something like `np.arange().reshape()` instead?